web/src/lib/stores/preferences.store.ts

```diff
@@ -22,6 +22,7 @@ export const locale = persisted<string | undefined>('locale', undefined, {
 export interface MapSettings {
   allowDarkMode: boolean;
   onlyFavorites: boolean;
+  seeArchived: boolean;
```

Comment:
What do you think about `withArchived` instead?

Reply:
can be an alternative! i try when come home!!!